distmaker/dists/common.sh

```diff
@@ -232,7 +232,7 @@ function dm_install_wordpress() {
     "$repo/./"  "$to/./"
   ## Need --exclude=civicrm for self-building on WP site
 
-  dm_preg_edit '/^Version: [0-9\.]+/m' "Version: $DM_VERSION" "$to/civicrm.php"
+  dm_preg_edit '/^([ \*]*)Version: [0-9\.]+/m' "\1Version: $DM_VERSION" "$to/civicrm.php"
   dm_preg_edit "/^define\( \'CIVICRM_PLUGIN_VERSION\',\W'[0-9\.]+/m" "define( 'CIVICRM_PLUGIN_VERSION', '$DM_VERSION" "$to/civicrm.php"
```

Comment:
@totten This regex will also need the whitespace removed since the CiviCRM-WordPress code is now `civilint` compatible.

Reply:
Good catch @christianwach. Tweaked to ignore the intervening spaces, and `r-run` seems fine on my local.